lib/ansible/galaxy/collection/__init__.py

```diff
@@ -329,41 +329,46 @@ def verify_local_collection(
 
 
 def verify_file_signatures(fqcn, manifest_file, detached_signatures, keyring, required_successful_count, ignore_signature_errors):
-    # type: (str, str, List[str], str, int, List[str]) -> bool
+    # type: (str, str, List[str], str, str, List[str]) -> bool
     successful = 0
     error_messages = []
 
+    strict = required_successful_count.startswith('+')
```

Comment:
If you did the regex thing above, you could probably also share this logic with the CLI validator

Reply:
Fixed. Thanks very much for the careful review, the state table gave me quite a headache :smile: